Gemfile

```diff
@@ -1,6 +1,10 @@
 source "https://rubygems.org"
 
-gem "xcode-install", ">= 2.0.10"
+gem "xcode-install", ">= 2.0.10" # needed for running xcode-install related tests
+
+gem "danger", ">= 4.2.1", "< 5.0.0"
+gem "danger-junit", ">= 0.7.3", "< 1.0.0"
+
```

Comment:
That would be the behavior you'd get with the two above, though it could be possible that another dependency could request danger `4.2.0` and it could resolve to that instead of `4.2.1` on a `bundle update x y`